eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -12,8 +12,8 @@ param (
   [String]$ServiceDirectory,
   [Parameter(Mandatory = $true)]
   [String]$PackageName,
-  [boolean]$Unreleased=$True,
-  [boolean]$ReplaceLatestEntry = $False,
+  [String]$Unreleased=$True,
+  [String]$ReplaceLatestEntryTitle = $False,
```

Comment:
What cases would you want to replace the title? Is it only when you want to update the version?

Reply:
Yes, its the same function as `ReplaceVersion ` in the current script. I just though this was a better title, since it not only replaces the version but also changes the "(ReleaseStatus)" in some cases